spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -45,8 +45,14 @@ def handle_two_step(response)
     def handle_two_factor(response)
       two_factor_url = "https://github.com/fastlane/fastlane/tree/master/spaceship#2-step-verification"
       puts "Two Factor Authentication for account '#{self.user}' is enabled"
-      puts "If you're running this in a non-interactive session (e.g. server or CI)"
-      puts "check out #{two_factor_url}"
+
+      if !File.exist?(persistent_cookie_path) && self.class.spaceship_session_env.to_s.length.zero?
```

Comment:
I know it has been a long time, but:
Does this line make any sense?

I read it as:

```
If `persistent_cookie_path` does not exist AND session env var is 0 characters long (= empty)
   show information
else  
   "Your session cookie has been expired."
```

Doesn't this also mean that as soon as the session env var is set, you _always_ get "Your session cookie has been expired." here? Or how does the checking for an expired session actually work?

Reply:
I don't this works at all - so reverted in https://github.com/fastlane/fastlane/pull/13743 where I also added a hopefully working check.